doc: gs: add updating page (#4338)
nrfconnect/sdk-nrf: doc/nrf/gs_installing.rst

    #. Copy-paste the license key and click :guilabel:`Install License`.
       The license activation window will close and SES will open the Project Explorer window.

.. note::

>     Notifications about newer versions of SES are disabled in SES Nordic Edition.

Reviewer: Is it the SES Nordic Edition or without the definite article? We have a few different version in this PR.

Author: I'll stick to "the" as per website: https://www.nordicsemi.com/Software-and-tools/Development-Tools/Segger-Embedded-Studio

Reviewer: So, SEGGER Embedded Studio, but the SEGGER Embedded Studio Nordic Edition.

>     This is because of the custom |NCS| options available only in the Nordic Edition.

Reviewer: ```suggestion
    This is because of the custom |NCS| options are available only in the Nordic Edition.
```

>     This is because of the custom |NCS| options available only in the Nordic Edition.
>     You won't be able to use SES with |NCS| if you install the latest standard version of SES.

Reviewer: ```suggestion
    This is because of the custom |NCS| options are available only in the Nordic Edition.
    You will not be able to use SES with |NCS| if you install the latest standard version of SES.
```

You must also make sure that nrfjprog (part of the `nRF Command Line Tools`_) is installed and its path is added to the environment variables.

> The west command programs the kit by using nrfjprog by default.

Reviewer: ```suggestion
The west command programs the development kit by using nrfjprog by default.
```
